pkgs/by-name/ko/komac/package.nix

```diff
@@ -44,6 +45,7 @@ rustPlatform.buildRustPackage {
       dbus
       openssl
       zstd
+      bzip2
```

Comment:
> added bzip since it is needed since 2.7.0

https://github.com/russellbanks/Komac/commit/727cf3b7fa88a2d9161029d8226f064015268631#diff-2e9d962a08321605940b5a657135052fbcef87b5e360662bb527c96d9a615542R33

Yeah, the bzip dependency was added in this commit. However, the nixpkgs build succeeded in 2.8.0 without this buildInputs. I don't know why 🤔

Reply:
bzip2 crate builds bzip2 from source if it is not available on the system.

Reply:
Understood! Thank you for describing it. 🙏